src/modules/core/chain_loader.ini

```diff
@@ -0,0 +1,9 @@
+# This file defines the chain normalizers, their fallbacks and the order they're attached
```

Comment:
I put this file in "core" so that it can be by the loader.ini file for producers. I am open minded about the name and location.

Also, I notice that the existing loader.ini is used in two places:

https://github.com/mltframework/mlt/blob/master/src/modules/core/producer_loader.c#L221

https://github.com/mltframework/mlt/blob/0f14c1406e986900ec40de3a7b101333dcf5db8f/src/modules/core/consumer_multi.c#L123

If we are interested, I am willing to move this code into a common function mlt_producer_attach_normalizers() to be similar to the new chain function. This would be a project for a different pull request. Let me know if you are interested.

Reply:
Not interested

Reply:
This is the wrong location because it is not used by the core module. It goes in the same location as metaschema.yaml

Reply:
Moved to framework and same location as metaschema.yaml.